### ISSUE TITLE
'advimage' plugin enhancement

### ISSUE THREAD

**Issue by TinyMCE-User** [tinymce/tinymce#1943]
Can the plugin be enhanced so that while inserting an image, there should be a decorative image check box, selecting it will allow the user to add image without the image description field entry. So as to resolve the accessibility errors

> _**Legacy information imported from TinyMCE bug tracker:**_
> 
> _[#T7214](http://archive.tinymce.com/develop/bugtracker_view.php?id=7214) posted by [rakshith](http://archive.tinymce.com/profile.php?name=rakshith)_
> 
> _**Tags:** []_
> _**Status:** Open_
> _**Resolution:** None_
> _**Attached URL:** none_

**Comment by fyrkant** [comment]
Closing this due to old age, if anyone still has this issue in the latest version of TinyMCE please open up a new issue.
